src/three-components/Renderer.ts

```diff
@@ -96,6 +102,10 @@ export class Renderer extends EventDispatcher {
       this.renderer.physicallyCorrectLights = true;
       this.renderer.setPixelRatio(resolveDpr());
 
+      if (options == null || options.debug === false) {
+        this.debugger = new Debugger(this);
```

Comment:
Is this condition backwards? Seems like you make a debugger if debug is false.

Reply:
Ah, yes, good catch. Thanks!

Reply:
Fixed